src/features/game/lib/processEvent.ts

```diff
@@ -122,7 +122,7 @@ export const MAX_ITEMS: Inventory = {
   Iron: new Decimal("800"),
   Stone: new Decimal("1600"),
   Wood: new Decimal("8000"),
-  "Wild Mushroom": new Decimal("75"),
+  "Wild Mushroom": new Decimal("100"),
```

Comment:
@eliasSFL, why did you revert my changes? I see a contradiction here. If it matters to keep a tolerance between FE and BE, the wild mushrooms should also follow this rule.
It's the same situation that players are experiencing with both kinds of mushrooms: they receive a sync message after getting the 100th wild mushroom and an error message after getting the 100th magic mushroom between two syncs. Your idea is to leave things as they are, which is also inconsistent with the rule that says to keep a tolerance between FE and BE. Even if we want to follow your idea, we should leave the magic mushrooms as they are too.